scripts/e2e-deploy.sh

```diff
@@ -121,19 +109,36 @@ echo "Adding registry.redhat.io secret."
 # the json distributed by access.redhat.com/terms-based-registry is a nested object.
 # oc wants the contents of the .data object, so we need to unwrap the outer layer
 # in order to load the pull secrets dockerconfigjson object into the secret.
-if [ -f ${REGISTRY_REDHAT_IO_SECRETS} ]; then
-    SECRET=$(cat ${REGISTRY_REDHAT_IO_SECRETS} | \
+if [ -f $REGISTRY_REDHAT_IO_SECRETS ]; then
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        # Mac OSX
+        SECRET=$(cat $REGISTRY_REDHAT_IO_SECRETS | \
+             python -c 'import yaml, sys; print(yaml.safe_load(sys.stdin).get("data").get(".dockerconfigjson"))' | \
+             base64 -D)
```

Comment:
The only difference I'm seeing here is the flag being passed to base64. Can we DRY this up a bit?